src/screens/Home/details.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+
+import BlankSlate from 'screens/Home/blankSlate';
+
+const Details = () => {
+  const date = new Date();
+  const date_options = { weekday: 'long', month: 'long', day: 'numeric' };
+  const today = date.toLocaleString('en-EN', date_options).toUpperCase();
+
+  return (
+    <div className="home-details">
```

Comment:
For class names, should we use the `singular` form instead?

Reply:
Fixed in [9792a50](https://github.com/rafayet-monon/surveyor/pull/15/commits/9792a50e0e309985ae60016ea580bf01c5cc701f)